airbyte-api/src/main/openapi/config.yaml

```diff
@@ -2432,6 +2460,13 @@ components:
       properties:
         slug:
           type: string
+    # Geography
+    Geography:
+      type: string
+      enum:
+        - auto
```

Comment:
lowercase vs capitalized, does it matter here? i think the database enums are capitalized. We should double check if the frontend has a preference on how it's returned to them